app/models/user.rb

```diff
@@ -1,5 +1,7 @@
 class User < ActiveRecord::Base
   validates :full_name, presence: true
+  validates_format_of :email, :with => /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/i, :on => :create
```

Comment:
Line is too long. [100/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer the new style validations `validates :column, format: value` over `validates_format_of`.